stock_release_channel/models/stock_picking.py

```diff
@@ -49,24 +51,44 @@ def _inverse_release_channel_id(self):
             all_moves = moves.browse(move_chain_ids)
             all_moves._release_set_expected_date(new_date)
 
-    @api.depends("release_channel_id", "need_release", "partner_id")
+    @api.depends(
```

Comment:
the logic looks good but I would like to move this stuff to where the triggers happen (eg: set the rel chan, change the state, release post hook, etc) as we discussed.